src/widgets/query-rule-context/__tests__/query-rule-context-test.ts

```diff
@@ -0,0 +1,18 @@
+import queryRuleContext from '../query-rule-context';
+
+describe('queryRuleContext', () => {
+  describe('Usage', () => {
+    test('does not throw without options', () => {
```

Comment:
do we not want it to throw without `trackedFilters`?

Reply:
That's a fair point – what's your thoughts @algolia/instantsearch-for-websites?

Reply:
Right now it's a noop in the connector so we can throw an error. 

But it means that we cannot track everything by default. Worth asking to the SE if it's a use case or not.

Reply:
We couldn't track everything by default anyway. The default behavior is to track nothing.

Besides, we don't recommend tracking everything because there's a limitation of 10 `ruleContexts` sent to the engine.

I will update the widget to throw if no `trackedFilters` are passed.